tests/Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cf
 RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
     ln -s /cache/images /build/images && \
     ln -s /cache/images /build/virt-builder && \
+    ln -s /cache/images /build/libvirt/qemu/log && \
```

Comment:
We could ... that's more complicated to work out. I'd like to consume them via http, but having them in a subdirectory isn't *that* bad. It does make the whole symlinking stuff more complicated though, having to create directories with correct permissions during startup etc.